pkgs/by-name/re/regrippy/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "regippy";
+  version = "2.0.1";
+  doCheck = false;
```

Comment:
Why disable checks? 
```suggestion
```

Since there is an upstream test suite, we may as well run it, using:
```nix
  nativeCheckInputs = [
    python3.pkgs.pytestCheckHook
  ];
```